scripts/install-mender.sh

```diff
@@ -443,6 +436,10 @@ check_dist_and_version() {
 
     echo "  Detected distribution:"
     printf "\t%s/%s\n" "$LSB_DIST" "$DIST_VERSION"
+
+    if [[ "$LSB_DIST" == "raspbian" ]]; then
```

Comment:
Suggestion:

Just a short log message here probably won't hurt us.

Reply:
I went a bit back and forth with this because I couldn't find a good way of logging it. But once you comment on it, means that I have to do a better effort :) I'll try